vhdl_lang/src/named_entity.rs

```diff
@@ -631,8 +632,11 @@ impl HasEntityId for Declaration {
             Declaration::SubprogramBody(body) => body.ent_id(),
             Declaration::SubprogramInstantiation(decl) => decl.ent_id(),
             Declaration::Package(pkg) => pkg.ent_id(),
+            Declaration::PackageDeclaration(pkg) => pkg.ent_id(),
+            Declaration::PackageBody(pkg) => pkg.ent_id(),
             Declaration::Use(_) => None,
             Declaration::Configuration(_) => None,
+            Declaration::Disconnection(_) => None, // TODO
```

Comment:
This would just return the `EntityId` of the disconnect specification. For simplicity, you can just implement it in a way that for an identifier list > 1, i.e., `disconnect a, b, c, ...`, a new `Disconnect` struct should be created.

Reply:
Done. But I still haven't figured out how to parse multiple idents, so there will be more to do here once I have.

Reply:
I did the multiple idents, so I think this is should be done now too, right?

Reply:
Yep, this looks nice!